odl/phantom/__init__.py

```diff
@@ -0,0 +1,39 @@
+﻿# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test data for tomography problems."""
+
+from __future__ import absolute_import
+
+__all__ = ()
+
+from . import phantom_utils
+
+from .emission_phantoms import *
+__all__ += emission_phantoms.__all__
+
+from .geometric_phantoms import *
+__all__ += geometric_phantoms.__all__
+
+from .misc_phantoms import *
+__all__ += misc_phantoms.__all__
```

Comment:
Don't think we need `_phantoms` for each module. It's pretty clear already from `odl.phantom.emission` what you get.